CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1840,7 +1846,15 @@ if gameOptions and not isTenFootInterface then
 end
 
 local function topBarEnabledChanged()
-	topbarEnabledChangedEvent:Fire(topbarEnabled)
+	if InputService.VREnabled then
+		lookMenuEnabled = topbarEnabled
+		topbarEnabled = false
```

Comment:
It should be okay if we think of it as transferring the state to lookMenuEnabled. I'll add preliminary support for toggling VR just so we don't forget to transfer back to topbarEnabled in the future.